contract-deployment/contractDeploymentTools.go

```diff
@@ -166,7 +41,7 @@ func main() {
 			log.Panic("Failed to parse nonce as uint", err)
 		}
 
-		resultAddress := deriveEVMContractAddress(deployerAddress, nonce)
+		resultAddress := deployment_utils.DeriveEVMContractAddress(deployerAddress, nonce)
```

Comment:
would prefer the utils function to return the error, then we handle the err here with panic